static_src/actions/org_actions.js

```diff
@@ -5,61 +5,72 @@
  */
 
 import AppDispatcher from '../dispatcher.js';
+import cfApi from '../util/cf_api.js';
 import { orgActionTypes } from '../constants';
 import spaceActions from './space_actions';
 
-export default {
+const orgActions = {
 
   changeCurrentOrg(orgGuid) {
     AppDispatcher.handleViewAction({
       type: orgActionTypes.ORG_CHANGE_CURRENT,
       orgGuid
     });
+
+    return Promise.resolve(orgGuid);
   },
 
   fetch(orgGuid) {
     AppDispatcher.handleViewAction({
       type: orgActionTypes.ORG_FETCH,
       orgGuid
     });
+
+    return cfApi.fetchOrg(orgGuid).then(orgActions.receivedOrg);
```

Comment:
I'm trying to avoid using `this` in the actions because it's possible the action will get passed as a function (like above) and when called, won't be bound to `this`. Explicitly referring to `orgActions` here means we never have to worry about `this` being unbound.